src/server/api/controllers/cvs.controller.js

```diff
@@ -4,18 +4,30 @@ const knex = require('../../config/db');
 const Error = require('../lib/utils/http-error');
 const moment = require('moment-timezone');
 
-const getCvs = async () => {
+const getCvs = async (title, limit) => {
   try {
-    return await knex('cv').select('cv.id', 'cv.title');
+    if (title) {
+      return await knex('cvs')
+        .select('cvs.id', 'cvs.title')
+        .where({ title });
```

Comment:
What happens if i search for a partial title. fx `skilled webdevel` then it wont find the cs `skilled webdeveloper` right? or maybe it will